### ISSUE TITLE
"Get started with WebView2 in WinUI 3" sample stops responding after opening downloaded file from the bottom bar

### ISSUE THREAD

**Issue by NoriyukiIchijo** [MicrosoftEdge/WebView2Feedback#1486]
Follow the instructions in
https://docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui
to make sample app, build the app, go to page with download link like
https://developer.microsoft.com/ja-jp/microsoft-edge/webview2/#download-section
download file and open it from the bottom bar.
Then app stops responding.

[AB#34372096](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/34372096)

**Comment by champnic** [comment]
Thanks for the report @NoriyukiIchijo! I've added this bug to our backlog to take a look.

**Comment by champnic** [comment]
Hey @NoriyukiIchijo - I was unable to repro this issue. Let me know if you are still running into this issue and we can take another look. Thanks!

**Comment by BioechonetDeveloper** [comment]
I noticed that the behavior of downloading file is changed as [#1420](https://github.com/MicrosoftEdge/WebView2Feedback/issues/1420).
But when I tried this new behavior, I found that "Download" panel was shown behind the window.
![20211125_113704](https://user-images.githubusercontent.com/90024410/143371326-ba088dff-354c-4bca-a165-0073eedfe156.gif)
I got to move the window to show the panel but once the panel gets focus, it behaves like modal dialog.
Can be only our case but this is a strange behavior.
